src/ui/components/Account/AddExternalAccount.tsx

```diff
@@ -137,7 +137,7 @@ export function AddExternalAccount({onClose}: Props) {
                 <Tabs style={{backgroundColor: colors.background}} onChangeIndex={(index) => setTabIndex(index)}>
                   <TabScreen label="Type in" icon="keyboard">
                     <View style={globalStyles.paddedContainer}>
-                      <BottomSheetTextInput
+                      <TextInput
```

Comment:
Since you are using the regular TextInput `useBottomSheetInternal` need to be used to make it work with keyboard. An example can be found in `SendFund.tsx`